benchmarks/TFLite/CMakeLists.txt

```diff
@@ -109,14 +109,134 @@ iree_benchmark_suite(
   TARGET_ARCHITECTURE
     "CPU-ARM64-v8A"
   TRANSLATION_FLAGS
-    "--iree-input-type=tosa"
-    "--iree-llvm-target-triple=aarch64-none-linux-android29"
+    "${ANDROID_CPU_TRANSLATION_FLAGS}"
   DRIVER
     "dylib"
   RUNTIME_FLAGS
     "--task_topology_group_count=1"
 )
 
+# CPU, Dylib, 2 through 4-thread, little-core, full-inference
```

Comment:
Hmm, these makes me wonder we should have another level of permutation for `RUNTIME_FLAGS` to avoid these explicit copies.. Gah. CMake is really not good at this..

(No request for change for now.)